source/_posts/2020-06-03-release-111.markdown

```diff
@@ -107,11 +110,8 @@ continue, resulting in a faster startup.
 - Add Blebox climate support ([@gadgetmobile] - [#35373]) ([blebox docs]) (new-platform)
 - Add lock platform to ozw component ([@mrk-its] - [#36103]) ([ozw docs]) (new-platform)
 - Add fan platform to ozw component ([@cgarwood] - [#35249]) ([ozw docs]) (new-platform)
-- Update plugwise to async and config_flow ([@CoMPaTech] - [#33691]) ([plugwise docs]) (breaking-change) (new-platform)
-- Update plugwise to async and config_flow sensor part ([@CoMPaTech] - [#36219]) ([plugwise docs]) (new-platform)
+- Update plugwise to async and config_flow, adding binary_sensor, sensor and switch ([@CoMPaTech], [@bouwew] - [#33691], [#36219], [#36378], [#36383]) ([plugwise docs]) (breaking-change) (new-platform)
 - Refactor Synology entries to allow not fetching the API when it's disabled + add security binary sensor ([@Quentame] - [#35565]) ([synology_dsm docs]) (breaking-change) (new-platform)
-- Update plugwise to async and config_flow binary_sensor part ([@CoMPaTech] - [#36378]) ([plugwise docs]) (new-platform)
-- Update plugwise to async and config_flow switch part ([@CoMPaTech] - [#36383]) ([plugwise docs]) (new-platform)
 - Add climate platform to Insteon ([@teharris1] - [#35763]) ([insteon docs]) (new-platform)
```

Comment:
These are commits and generated, please revert this change.

Reply:
Reverting, makes sense why Martin is always making sure the titles are optimal :)